Dockerfile.d/SHA256SUMS.d/cni-plugins-v1.4.1

```diff
@@ -1,2 +1,2 @@
-1511f6c003ace805eafeb1132727791326283cff88a923d76329e1892bba7a10  cni-plugins-linux-amd64-v1.4.1.tgz
-72644e13557cda8a5b39baf97fc5e93d23fdf7baba7700000e7e9efd8bdf9234  cni-plugins-linux-arm64-v1.4.1.tgz
+2a0ea7072d1806b8526489bcd3b4847a06ab010ee32ba3c3d4e5a3235d3eb138  cni-plugins-linux-amd64-v1.4.1.tgz
+56fe62d73942cffd8f119d2b8ecb6a062e85f529a3dbfc7aa5cd83c2c01929a7  cni-plugins-linux-arm64-v1.4.1.tgz
```

Comment:
Probably CNI plugins (and also other components) should be just updated to the latest version

Reply:
Will create a separate profile to update this branch and then do the cherry pick. Thanks for the comments

Reply:
What do you mean by "profile"?

Reply:
Sorry that was auto from the keyboard, I meant pr